src/main/java/ch/uzh/ifi/hase/soprafs24/service/SessionService.java

```diff
@@ -125,4 +127,20 @@ public Session endSession(Long teamId) {
         endedSession.getStartDateTime(), endedSession.getEndDateTime());
     return endedSession;
   }
+
+  private boolean isSessionExpired(Session session) {
+    return session.getStartDateTime().plusHours(24).isBefore(LocalDateTime.now());
+  }
+
+  @Scheduled(fixedRate = 60000) // runs every minute
```

Comment:
every minute is too expensive. make every hour

Reply:
Changed it to:
@Scheduled(fixedRate = 3600000)
@5v3nn